common/persistence/nosql/nosqlplugin/cassandra/admin.go

```diff
@@ -43,7 +42,7 @@ const (
 var _ nosqlplugin.AdminDB = (*cdb)(nil)
 
 func (db *cdb) SetupTestDatabase(schemaBaseDir string) error {
-	err := createCassandraKeyspace(db.session, db.cfg.Keyspace, 1, true)
+	err := db.createCassandraKeyspace(db.session, db.cfg.Keyspace, 1, true)
```

Comment:
probably worth changing this and `drop*` to use the fields rather than args - the args don't seem to be customized anywhere and there are only like two uses total.